src/pages/resultsView/ResultsViewPage.tsx

```diff
@@ -91,7 +94,56 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
     componentDidMount(){
 
+    }
+
+    componentDidUpdate() {
+        this.addBookmarkButtons();
+    }
+
+    public addBookmarkButtons() {
+        let shareURL = window.location.href;
+
+        if (shareURL.indexOf("session_id") < -1) return;
+
+        const showSamples = shareURL.indexOf("&show");
```

Comment:
you should really use a url parser for this.  we have one installed already called :"url"